frontend/src/components/details/DetailsComponent.js

```diff
@@ -1,24 +1,54 @@
 /**
- * the main component for detail page which includes  
- * canvas page and variable drop down list  
- * 
- * @author parham, 08.05  
+ * the main component for detail page which includes
+ * canvas page and variable drop down list
+ *
+ * @author parham, 08.05
  */
 
-import React, {Component, useContext} from 'react';
+import React, { Component, useContext } from 'react';
 import Canvas from './CanvasComponent';
 import SelectVariable from './SelectVariableComponent';
-import {PrivateSectionContext} from "hooks/PrivateSectionContext";
+import { PrivateSectionContext } from 'hooks/PrivateSectionContext';
+import DividerPannel from './PanelComponent';
+
+class DetailComponent extends Component {
```

Comment:
I don't wan't to be that guy, but ... The filename is still different 🤣

Reply:
I thought you won't realize it